source/features/latest-tag-button.tsx

```diff
@@ -72,7 +107,9 @@ async function init(): Promise<false | void> {
 		return false;
 	}
 
-	breadcrumbs.before(getTagLink(latestTag));
+	const [tagLink] = await Promise.all([getTagLink(latestTag)]);
+
+	breadcrumbs.before(tagLink);
```

Comment:
At the end, after all the suggestions I made, you should be able to revert this change and just use the previous one:

```js
breadcrumbs.before(getTagLink(latestTag)); 
```

`getTagLink(latestTag, isAhead)` should only generate the link, it doesn't need to be `async`